lib/mox.ex

```diff
@@ -226,8 +238,15 @@ defmodule Mox do
         :error -> raise ArgumentError, ":for option is required on defmock"
       end
 
+    skip_optional_callbacks =
+      case Keyword.get(options, :skip_optional_callbacks, []) do
+        skip_list when is_list(skip_list) -> skip_list
```

Comment:
Let's also allow the value to be `true` or `false`, which means skipping all or none!

Reply:
Okay, can do. I will update the PR to include this :)

Reply:
Okay, updated!